core-bundle/src/Picker/FilePickerProvider.php

```diff
@@ -36,7 +36,7 @@ class FilePickerProvider extends AbstractInsertTagPickerProvider implements DcaP
      */
     private $uploadPath;
 
-    public function __construct(Security $security, FactoryInterface $menuFactory, RouterInterface $router, TranslatorInterface $translator, string $uploadPath)
+    public function __construct(Security $security, string $uploadPath, FactoryInterface $menuFactory, RouterInterface $router, TranslatorInterface $translator = null)
```

Comment:
I don't think we should make the translator optional here. It was not optional to begin with, and the parent method triggers a warning if it is…

Reply:
Fixed in 11e0736891716a1fe301c14039f13fc6694c64ac.